src/styles/variables.scss

```diff
@@ -4,13 +4,17 @@ $primary: #ff6900;
 $secondary-light: #d8d8d8;
 $secondary-dark: #4d4d4d;
 $success: #009639;
+$highlight-data: #009639;
 $border-light: #eee;
 $light-box-bg: #f6f6f6;
 $dark-box-bg: #c6c6c6;
+$section-title: #4c5459;
+$health-icon: #7e7e7e;
 
 // Border 
 
 $smooth-border: 4px;
+$smooth: 4px;
```

Comment:
16 and 17 are the same?

Reply:
smooth-border is used for the width, smooth for the radius